### PR TITLE
Mock API updates

### PR DESCRIPTION
**Pull request by cdcabrera**
Additions for running a mock API from Swagger.
- Additional NPM script
- Bash updates

Part of #240

### CODE REVIEW

File: client/scripts/api.sh [hunk 1]
```diff
@@ -16,17 +17,25 @@
       esac
   done
 
-  if [ -z "$(which docker)" ]; then
-    echo "Setting up API Docker container"
-    docker pull $CONTAINER
+  if [ ! -z "$(which docker)" ]; then
```
Comment (jeff-phillips-18) [inline client/scripts/api.sh]: Is the ! correct here?

Comment (cdcabrera) [reply client/scripts/api.sh]: I'll double check

Comment (cdcabrera) [reply client/scripts/api.sh]: I'll double check

Comment (jeff-phillips-18) [reply client/scripts/api.sh]: That's pretty funny 😆

**Pull request merged**